Index.html

```diff
@@ -213,6 +213,16 @@ <h2>About Myself</h2>
             </li>
           </ul>
           <hr>
-</section>    
+</section>
+<section class="contact">   
+  <h1 >Contact me</h1>
+  <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
+  <form action="https://formspree.io/f/mqkoywbg" method="POST" class="pure-form" name="">            
+      <input type="text" class="input-rounded" placeholder="Lorem Ipsum" maxlength="30" name="name" required>                          
+      <input  type="email" class="input-rounded" placeholder="loremipsum@gmail.com" required>             
+      <textarea name="message" maxlength="500" placeholder="Write your message here"  required></textarea>
+      <button type="submit" class="call-to-action">Get in touch</button>
+  </form>
+</section>
```

Comment:
- [ ] Your form is working fine and has a similar UI from the Figma template. But at the bottom, the form section seems to end after the `textarea` and the button seems to stay out of it, in an area with a grey background color. Kindly, make the contact section end at the bottom of the screen, and make sure that the buttom seems to stay inside the contact section which has a purple background color.
 - Your design:
 - 
<img width="480" alt="image" src="https://user-images.githubusercontent.com/61919617/229815321-878a465c-1f98-4659-ad00-c8b1ccc84e1c.png">

- Figma template:
- 
<img width="280" alt="image" src="https://user-images.githubusercontent.com/61919617/229815520-de1d4327-5a99-45f9-baa2-0782b9eaef6a.png">